admin_guide/managing_networking.adoc

```diff
@@ -461,13 +456,11 @@ source IP address and using the specified gateway, you can specify a
 [[admin-guide-manage-pods-egress-router-multi-destination]]
 ==== Redirecting to Multiple Destinations
 
-In the example above, connections to the egress pod (or its
-corresponding service) on any port will be redirected to a single
-destination IP. It is also possible to configure different destination
-IPs depending on the port:
+In the example above, connections to the egress pod (or its corresponding
```

Comment:
Do not use to indicate a relative location in a document. Use "previous" or "preceding."